portrait_hw/index.js

```diff
@@ -0,0 +1,76 @@
+const userBirthYear = Number(prompt ('Enter your year of birth:'));
+let userCity = prompt('Enter the city where you live:');
+let userFavoriteSport = prompt('Enter your favorite kind of sport:');
+
+userCity = userCity.substr(0,1).toUpperCase() + userCity.substr(1).toLowerCase();
+userFavoriteSport = userFavoriteSport.toLowerCase();
+
+let thisDay = new Date();
```

Comment:
do you really need this variable

Reply:
I thought, if I should take a year, I should take it from the full date, || not? I'll repeat the information about  working with the date